aten/src/ATen/native/cpu/GridSamplerKernel.cpp

```diff
@@ -0,0 +1,847 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/Parallel.h>
+#include <ATen/core/C++17.h>
+#include <ATen/TensorUtils.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/native/GridSampler.h>
+#include <ATen/native/cpu/GridSamplerKernel.h>
+#include <ATen/cpu/vml.h>
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+#include <algorithm>
+#include <cstring>
+#include <type_traits>
+
+namespace at { namespace native { namespace {
+
+/**  NOTE [ Grid Sample CPU Kernels ]
```

Comment:
I think this comment would be even clearer if you gave an example!

Reply:
Example added (actually moved from near end of the note to beginning)!